e2e_samples/parking_sensors/scripts/deploy_appservice.sh

```diff
@@ -0,0 +1,27 @@
+export appName=data-simulator-$DEPLOYMENT_ID
+
+arm_output=$(
+    az deployment group create \
+  --resource-group $resource_group_name \
+  --template-file "./infrastructure/modules/appservice.bicep" \
+  --parameters appName=$appName \
+  --output json
+)
+
+if [[ -z $arm_output ]]; then
+    echo >&2 "ARM deployment failed."
+    exit 1
+fi
+
+az webapp config appsettings set --name $appName --resource-group $resource_group_name --settings WEBSITE_NODE_DEFAULT_VERSION="~16"
+az webapp deploy --resource-group $resource_group_name --name $appName --type zip --src-path ./data/data-simulator.zip
+
+export API_BASE_URL="https://$appName.azurewebsites.net"
```

Comment:
Can you please review the exports and follow the way we do it with the other parameters?